js/local-storage.js

```diff
@@ -0,0 +1,23 @@
+const fields = document.querySelectorAll('.field-data');
+
+let formData = {
+  name: '',
+  email: '',
+  message: '',
+};
+
+fields.forEach((field) => {
+  field.addEventListener('input', () => {
+    formData[field.name] = field.value;
+    localStorage.setItem('portfolio-form-data', JSON.stringify(formData));
+  });
+});
+
+window.addEventListener('load', () => {
+  const storedData = localStorage.getItem('portfolio-form-data');
+  formData = JSON.parse(storedData);
+
+  fields.forEach((field) => {
+    field.value = formData[field.name];
+  });
+});
```

Comment:
- I don't think data entered in your form is being saved to the local storage.  Can we fix this?? 🤗 

Here is what i could see on my browser after cloning your repo 👇🏽 

![empt](https://user-images.githubusercontent.com/99246359/209363690-4054217e-d3f0-4191-bb84-ac3fa74731d3.PNG)